library/farmosUtil/farmosUtil.transaction.js

```diff
@@ -28,6 +28,71 @@
  * If an operation was never attempted (and thus also not undone) the attribute for that operation will be `undefined`.
  *
  * @category Utilities
+ * 
+ * Executes a transaction with the provided operations.
+ *
+ * @param {Array<Object>} operations the operations to execute as a transaction.
+ * Each operation must have the following structure:
+ *
+ * ```
+ * {
+ * name: string,
+ * do: (Object) => async function performs an action and returns a result (e.g. creates a log, asset or quantity.) The argument has the same format as the return value. It contains the result of all prior operations, allowing them to be used by future operations.
+ * undo: (Object) => async function that undoes the action performed by `do` (e.g. deletes a log, asset or quantity). The argument has the same format as the return value and contains the result of all successfully completed operations.
+ * }
+ * ```
+ *
+ * @example
+ * // Create a transaction with a plant asset, a quantity, and a log
+ * const operations = [
+ *   {
+ *     name: 'createPlantAsset',
+ *     do: async () => {
+ *       return await farmosUtil.createPlantAsset(
+ *         '2023-10-01', // Example date
+ *         'ZUCCHINI',   // Example crop name
+ *         'Planting zucchini in the greenhouse' // Example comment
+ *       );
+ *     },
+ *     undo: async (results) => {
+ *       await farmosUtil.deletePlantAsset(results['createPlantAsset'].id);
+ *     },
+ *   },
+ *   {
+ *     name: 'createQuantity',
+ *     do: async () => {
+ *       return await farmosUtil.createStandardQuantity(
```

Comment:
This does not look like a valid call to the `createStandardQuantity` function.  You can find that function in `library/farmosUtil/farmosUtil.quantity.js` and you can find its described in the documentation.  The documentation is generated as part of the install process for the development environment as described in `INSTALL.md`. You can also see a valid call for an example in `modules/farm_fd2/src/entrypoints/tray_seeding/lib.js` where it creates the `traysQuantity`starting on line 38.